lib/mix/tasks/phx.gen.context.ex

```diff
@@ -99,7 +99,7 @@ defmodule Mix.Tasks.Phx.Gen.Context do
     {opts, parsed, _} = parse_opts(args)
     [context_name, schema_name, plural | schema_args] = validate_args!(parsed)
 
-    table = Keyword.get(opts, :table, Phoenix.Naming.underscore(context_name) <> "_#{plural}")
+    table = Keyword.get(opts, :table, (Phoenix.Naming.underscore(context_name) |> String.replace("/","_")) <> "_#{plural}")
```

Comment:
I did a cleaner solution. I hope this will do it. Thanks for the suggestion José.